packages/common/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@gomomento/core",
+  "name": "@gomomento/common",
```

Comment:
@bruuuuuuuce 
I wasn't sure it was intentionally named `core` although the folder is named `common`?